Makefile

```diff
@@ -359,9 +359,7 @@ endef
 # get-host-alias-ip switch to second context ($2), search for IP and switch back to first context ($1)
 # function returns one IP address
 define get-host-alias-ip
-	kubectl config use-context $2 > /dev/null && \
-	kubectl get nodes $2-agent-0 -o custom-columns='IP:status.addresses[0].address' --no-headers && \
-	kubectl config use-context $1 > /dev/null
+	kubectl --context $2 get nodes $2-agent-0 -o custom-columns='IP:status.addresses[0].address' --no-headers
```

Comment:
Argument $1 dismissed while $2 is used. Remove second argument and use the first only.

Reply:
done